pkg/util/parsers/parsers_test.go

```diff
@@ -24,10 +24,11 @@ import (
 // https://github.com/docker/docker/commit/4352da7803d182a6013a5238ce20a7c749db979a
 func TestParseImageName(t *testing.T) {
 	testCases := []struct {
-		Input  string
-		Repo   string
-		Tag    string
-		Digest string
+		Input         string
+		Repo          string
+		Tag           string
+		Digest        string
+		expectedError bool
```

Comment:
make the expected error a string and check the resulting error contains the expected error

Reply:
done.